<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtChatClass</class>
 <widget class="QMainWindow" name="QtChatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtChat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leIP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Порт сервера:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePort">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1213</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tbMain">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>200</number>
    </property>
    <property name="tabStopDistance">
     <double>200.000000000000000</double>
    </property>
    <property name="cursorWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButt">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLogin">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP сервера:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tbMess">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtChat.qrc"/>
 </resources>
 <connections/>
</ui>
